keps/sig-testing/2539-continuously-deploy-k8s-prow/README.md

```diff
@@ -114,36 +118,59 @@ One possible way of dealing with breaking changes, is:
 
 This approach uses tide auto-merge feature, so that no need to worry about repo requirements such as need more than one approver etc.
 
-```
-<<[UNRESOLVED (spiffxp) ]>>
-Suggestion: how to keep slack reports on each automated bump.
-<<[/UNRESOLVED]>>
-```
-
 #### Roll Back Process
 
 When prow stopped functioning after a bump, prow oncall should:
 - Stop auto-deploying by commenting `/hold` on latest autobump PR.
 - Manually create rollback PR for rolling back to known good version.
-- Manually apply the changes from rollback PR.
+  - Prow is not super actively developed currently, normally there are not many
+    changes between bumps, and it should be easy to identify culprit.
+  - General rule of thumb is we can assume last bump was good.
+- Manually apply the changes from rollback PR by running [`prow/bump.sh`](https://github.com/kubernetes/test-infra/blob/master/prow/deploy.sh)
+
+### Graduation Criteria
+
+#### Alpha -> Beta Graduation
+
+- Low frequency continuous deployment bumped prow as expected
+- Known prow failures are captured by alerts ahead of non-oncall human
+
+#### Beta -> GA Graduation
 
-```
-<<[UNRESOLVED]>>
-Which version to roll back. This is generally not a problem due to low release volume of prow. @alvaroaleman suggested 6 hours intervals.
-<<[/UNRESOLVED]>>
-```
+- High frequency continuous deployment bumped prow as expected
+- Testgrid displays prow plank version
+
+**Note:** Generally we also wait at least two releases between beta and
+GA/stable, because there's no opportunity for user feedback, or even bug reports,
+in back-to-back releases.
```

Comment:
Two releases of k8s? This KEP isn't tracked as part of the k8s release, so I'm not sure if this makes sense

Reply:
Agreed, bad copy/paste removed